packages/desktop-electron/window-state.ts

```diff
@@ -72,15 +83,18 @@ function hasBounds(state) {
   );
 }
 
-function validateState(state) {
-  if (!(hasBounds(state) || state.isMaximized || state.isFullScreen)) {
+function validateState(state?: WindowState): Partial<WindowState> {
+  if (
+    !state ||
+    !(hasBounds(state) || state.isMaximized || state.isFullScreen)
+  ) {
     return {};
   }
 
   const newState = Object.assign({}, state);
 
   if (hasBounds(state) && state.displayBounds) {
-    const screen = electron.screen || electron.remote.screen;
```

Comment:
electron.remote.screen is deprecated and the alternative is not the recommended appraoch. We should be safe to remove it.

![image](https://github.com/actualbudget/actual/assets/5285928/80f3cf18-e90d-42a8-8c9f-103a3788b6b8)